openlane_run/action.yml

```diff
@@ -0,0 +1,31 @@
+name: Run design with latest OpenLane release
+
+inputs:
+  design_name:
+    description: Name of design to run
+    required: true
+  pdk_root:
+    description: PDK root path
+    required: true
+  ol_tag:
+    description: OpenLane tag
+    default: latest
+
+
+runs:
+  using: composite
+  steps:
+
+    - name: Export PDK ROOT
+      shell: bash
+      run: |
+        echo "PDK_ROOT="${{ inputs.pdk_root }} >> $GITHUB_ENV
```

Comment:
giving that those are directly derived from the `inputs`, I'm curious if we could define those as regular `env` keys https://docs.github.com/en/actions/learn-github-actions/environment-variables#about-environment-variables here?

Reply:
I was just following the "style" on the repo and other actions on the organization. But I will take look at if env can resolve a `${{ input.xyz }}` statement.